packages/style-engine/docs/using-the-style-engine-with-block-supports.md

```diff
@@ -187,29 +190,43 @@ print_r( $styles );
 
 /*
 array(
-    'css'        => 'background-color:var(--wp--preset--color--blue);font-size:var(--wp--preset--font-size--small);',
-    'classnames' => 'has-background-color has-blue-background-color has-small-font-size',
+    'css'          => 'background-color:var(--wp--preset--color--blue);padding:10px;margin-top:1em;font-size:var(--wp--preset--font-size--small);',
+	'declarations' => array(
+		'background-color' => 'var(--wp--preset--color--blue)',
+		'padding' => '10px',
+		'margin-top' => '1em',
+		'font-size' => 'var(--wp--preset--font-size--small)',
+	),
+    'classnames'   => 'has-background has-blue-background-color has-small-font-size',
 )
 */
 ```
 
-If you don't want the Style Engine to output the CSS custom vars as well, which you might not if you're applying both the CSS and classnames to the block element, you can pass `'convert_vars_to_classnames' => true` in the options array.
+If you don't want the Style Engine to output the CSS custom vars in the generated CSS string as well, which you might not if you're applying both the CSS rules and classnames to the block element, you can pass `'convert_vars_to_classnames' => true` in the options array.
+
+This flag means "covert the vars to classnames and don't output the vars to the CSS". The Style Engine will therefore **only** generate the required classnames and omit the CSS custom vars in the CSS.
+
+Using the above example code, observe the different output when we pass the option:
 
 ```php
 $options = array(
-	// Whether to skip converting CSS var:? values to var( --wp--preset--* ) values. Default is `false`.
-	'convert_vars_to_classnames' => 'true',
+	'convert_vars_to_classnames' => true,
 );
+
 $styles = wp_style_engine_get_styles(
-	$block_styles,
-	$options
+    $block_styles,
+    $options
 );
 print_r( $styles );
 
 /*
 array(
-    'css'        => 'letter-spacing:12px;', // non-preset-based CSS will still be compiled.
-    'classnames' => 'has-background-color has-blue-background-color has-small-font-size',
+    'css'          => 'padding:10px;margin-top:1em;',
```

Comment:
Tiniest of nits: this line appears to use spaces to indent whereas some of the lines below use tabs.